### ISSUE TITLE
Hyperparameter selection problems when transferring diffusion models

### ISSUE THREAD

**Issue by zezeze97** [Derekabc/DMWeeds#2]
Hello, when I tried to replicate your transfer diffusion experiment using OpenAI's open source guided diffusion, I don't know how to choose batch size and microbatch. There is definitely no way to use a batch size of 128 in the case of a single GPU, and your LR will also be reduced in the case of reducing the batch size?

**Comment by DongChen06** [comment]
Hi @zezeze97, I tried batch sizes like 2, 4, and 8 for 3090 Ti. you may have to try the smallest batch size to see if it works.

**Comment by zezeze97** [comment]
Is there an adjustment to the LR during diffusion transfer learning, or is the default 1e-4 used?

**Comment by DongChen06** [comment]
Hi, I did not change the learning rate.
